File: Data Structures & Algorithms/find-target-in-rotated-sorted-array/submission-2.java
class Solution {
    public int search(int[] nums, int target) {
        int r = nums.length-1;
        int l = 0;
        int res = -1;

        while(l <= r){
            int m = (l+r) / 2;
            if(nums[m] > nums[r] && target < nums[m]){
                l = m+1;
            }
            else if(nums[m] < nums[r] && target > nums[m]){
                l = m+1;
            }
            else{ 
                 r = m-1;
                 System.out.println(nums[m]);
                if(nums[m] == target){
                    res = m;
                }
            }

        }

        return res; 
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-2.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {

        List<int> results = new List<int>();

        for (int i=0; i<nums.Length;i++){
            for(int j=0; j<nums.Length;j++){
                if (nums[i] + nums[j] == target){
                    results.Add(i);
                    results.Add(j);
                    i = nums.Length;
                    j = nums.Length;
                }
            }

        }

        return results.ToArray();

    }
}
